.buildkite/Dockerfile-compile

```diff
@@ -1 +1 @@
-FROM public.ecr.aws/docker/library/golang:1.22.3@sha256:f43c6f049f04cbbaeb28f0aad3eea15274a7d0a7899a617d0037aec48d7ab010
+FROM public.ecr.aws/docker/library/golang:1.21.11
```

Comment:
pact-go doesn't work with go 1.22.x at the moment.
https://github.com/pact-foundation/pact-go/issues/402

> Linux x86_64 users
>
> Using GoLang 1.22.x ?
> Resolution: disable -race
> Need to use  -race?
> Resolution: downgrade to 1.21.x